graphql/resolvers.go

```diff
@@ -250,6 +250,11 @@ func (r *taskResolver) Project(ctx context.Context, obj *restModel.APITask) (*re
 	return &apiProjectRef, nil
 }
 
+func (r *taskResolver) ProjectIdentifier(ctx context.Context, obj *restModel.APITask) (*string, error) {
```

Comment:
Why don't we do this in buildFromService so that we don't need this resolver?

Reply:
This performs a db query to resolve. We do not always need this field. So it becomes a wasted db query.
BuildFromService should not really run any db queries anyways it is meant to convert one struct into another. By having a db query it slows that conversion down dramatically.

Reply:
wouldn't buildFromService have it without needing a query?

Reply:
oh, never mind. this is the task resolver.